setup.py

```diff
@@ -171,7 +171,7 @@ def run(self):
 
 setuptools_args = {}
 install_requires = setuptools_args['install_requires'] = [
-    'mistune',
+    'mistune!=0.6',
```

Comment:
Just checked with 0.5.1: passing `escape=False` should work, I think just the default changed from False to True.